### ISSUE TITLE
Hardcoded label in Accessibility App details

### ISSUE THREAD

**Issue by kevgathuku** [nextcloud/server#10066]
### Steps to reproduce
1. Navigate to Apps > Click on `Accessibility` app to open Details view

### Expected behaviour
The label should be translated and not hardcoded

### Actual behaviour
![output 1](https://user-images.githubusercontent.com/3721994/42132858-8bf6d04a-7d28-11e8-9b15-f3ff69ebcea6.png)

### Server configuration

**Nextcloud version:** 14.0.0 alpha
